kiliautoml/models/_base_model.py

```diff
@@ -40,7 +40,7 @@ def train(
         disable_wandb: bool,
         verbose: int,
         **kwargs,
-    ) -> DictTrainingInfos:
+    ) -> Dict[str, Any]:
```

Comment:
You should keep `DictTrainingInfos` and define it as `DictTrainingInfos= Dict[str, Any]`